panoramisk/ami_protocol.py

```diff
@@ -50,8 +50,11 @@ def data_received(self, data):
         # self.log.debug('data received: "%s"', data)
 
         if self.version is None:
-            if data.startswith('Asterisk Call Manager/'):
-                self.version = data.strip().split('/')[1]
+            if data.startswith(b'Asterisk Call Manager/'):
```

Comment:
I dont really understand why you're using bytes. data are already decode at https://github.com/ssash/panoramisk/blob/53d1ff82caf46e7fd3b188edce4cd7015ba047e2/panoramisk/ami_protocol.py#L41

Reply:
Oops! It's my mistake.
I'll correct this.